tools/stern/test/integration/API.hs

```diff
@@ -93,14 +95,34 @@ tests s =
       test s "GET /i/consent" testGetConsentLog,
       test s "GET /teams/:id" testGetTeamInfo,
       test s "GET i/user/meta-info?id=..." testGetUserMetaInfo,
-      test s "/teams/:tid/search-visibility" testSearchVisibility
+      test s "/teams/:tid/search-visibility" testSearchVisibility,
+      test s "i/oauth/clients" testCrudOAuthClient
       -- The following endpoints can not be tested because they require ibis:
       -- - `GET /teams/:tid/billing`
       -- - `GET /teams/:tid/invoice/:inr`
       -- - `PUT /teams/:tid/billing`
       -- - `POST /teams/:tid/billing`
     ]
 
+testCrudOAuthClient :: TestM ()
+testCrudOAuthClient = do
```

Comment:
i won't insist, but maybe add new stern tests to `/integration` as well in the future?  it's also good to save time, though...

Reply:
Yes, good point. Now the work is done. Let's keep it for now.